WooCommerce/Classes/ViewRelated/Blaze/BlazeIntro/BlazeCreateCampaignIntroView.swift

```diff
@@ -0,0 +1,228 @@
+                            .fontWeight(.semibold)
+                            .headlineStyle()
+
+                        Spacer()
+                    }
+
+                    VStack(spacing: Layout.titleElementsVerticalSpacing) {
+                        Text(Localization.title)
+                            .fontWeight(.bold)
+                            .largeTitleStyle()
+                            .multilineTextAlignment(.center)
+
+                        Text(Localization.subtitle)
+                            .foregroundColor(Color(.secondaryLabel))
+                            .bodyStyle()
+                            .multilineTextAlignment(.center)
+
+                        Image(uiImage: UIImage.blazeIntroIllustration)
+                    }
+
+                    VStack(spacing: Layout.featuresVerticalSpacing) {
+                        ForEach(features, id: \.title) { feature in
+                            FeatureView(feature: feature)
+                        }
+                    }
+                }
+                .padding(Layout.contentPadding)
+            }
+            .safeAreaInset(edge: .bottom) {
+                VStack(spacing: Layout.CTAStack.spacing) {
+                    Divider()
+                        .frame(height: Layout.CTAStack.dividerHeight)
+                        .foregroundColor(Color(.separator))
+
+                    Button(Localization.createYourCampaign) {
+                        onCreateCampaign()
+                    }
+                    .buttonStyle(PrimaryButtonStyle())
+                    .padding(.horizontal, Layout.CTAStack.buttonHPadding)
+
+                    Button(Localization.learnHowBlazeWorks) {
+                        onCreateCampaign()
+                    }
+                    .buttonStyle(LinkButtonStyle())
+                    .padding(.horizontal, Layout.CTAStack.buttonHPadding)
+                }
+                .background(Color(UIColor.systemBackground))
+            }
+        }
+    }
+}
+
+private extension BlazeCreateCampaignIntroView {
+    struct Feature {
+        let title: String
+        let subtile: String
+    }
+
+    struct FeatureView: View {
+        /// Scale of the view based on accessibility changes
+        @ScaledMetric private var scale: CGFloat = 1.0
+
+        let feature: Feature
+
+        var body: some View {
+            HStack {
+                VStack(alignment: .leading) {
+                    Image(uiImage: .blaze)
+                        .resizable()
+                        .renderingMode(.template)
+                        .foregroundColor(Color(.brand))
+                        .frame(width: Layout.logoSize * scale, height: Layout.logoSize * scale)
+
+                    Text(feature.title)
+                        .fontWeight(.bold)
+                        .bodyStyle()
+
+                    Text(feature.subtile)
+                        .bodyStyle()
+                }
+
+                Spacer()
```

Comment:
I had to add this Spacer and HStack to get the desired horizontal padding. 

| Without Spacer | With Spacer |
|--------|--------|
| ![Simulator Screenshot - iPhone 14 Pro - 2023-12-27 at 21 17 37](https://github.com/woocommerce/woocommerce-ios/assets/524475/b4274693-ce5b-4040-b310-971708e258a3) | ![Simulator Screenshot - iPhone 14 Pro - 2023-12-27 at 21 16 18](https://github.com/woocommerce/woocommerce-ios/assets/524475/f48d7581-3ab7-4506-a065-f623d232c6fc) | 

Any suggestions are welcome!

Reply:
You can remove the HStack and Spacer and instead let the subtitle stretch to full width:

```Swift
                 Text(feature.subtile)
                        .bodyStyle()
                        .frame(maxWidth: .infinity, alignment: .leading)
```

Reply:
Thank you! Updated in 56c07341cd19959cdf3c5a3f140f95b3d6bae1c7